packages/encoder/test/wrap-fc.test.ts

```diff
@@ -277,48 +346,81 @@ describe("Wrapping of elementary values", () => {
       await encoder.wrapElementaryValue(info.type, info.input)
     );
     assert.strictEqual(wrapped.value.kind, "valid");
-    assert.strictEqual((<Codec.Format.Values.StringValueInfoValid>wrapped.value).asString, info.value);
+    assert.strictEqual(
+      (<Codec.Format.Values.StringValueInfoValid>wrapped.value).asString,
+      info.value
+    );
   });
 
   testProp("Bytes (static-length)", [BytesStaticInfo()], async info => {
     const wrapped = <Codec.Format.Values.BytesValue>(
       await encoder.wrapElementaryValue(info.type, info.input)
     );
-    assert.strictEqual(wrapped.value.asHex, Codec.Conversion.toHexString(info.value).padEnd(2 + 2 * info.type.length, "0"));
+    assert.strictEqual(
+      wrapped.value.asHex,
+      Codec.Conversion.toHexString(info.value).padEnd(
+        2 + 2 * info.type.length,
+        "0"
+      )
+    );
   });
 
   testProp("Bytes (dynamic-length)", [BytesDynamicInfo()], async info => {
     const wrapped = <Codec.Format.Values.BytesValue>(
       await encoder.wrapElementaryValue(info.type, info.input)
     );
-    assert.strictEqual(wrapped.value.asHex, Codec.Conversion.toHexString(info.value));
+    assert.strictEqual(
+      wrapped.value.asHex,
+      Codec.Conversion.toHexString(info.value)
+    );
   });
 
   testProp("Addresses", [AddressInfo()], async info => {
     const wrapped = <Codec.Format.Values.AddressValue>(
       await encoder.wrapElementaryValue(info.type, info.input)
     );
-    assert.strictEqual(wrapped.value.asAddress, Web3Utils.toChecksumAddress("0x" + info.value));
+    assert.strictEqual(
+      wrapped.value.asAddress,
+      Web3Utils.toChecksumAddress("0x" + info.value)
+    );
   });
 
   testProp("Unsigned decimals", [UfixedInfo()], async info => {
     const wrapped = <Codec.Format.Values.UfixedValue>(
       await encoder.wrapElementaryValue(info.type, info.input)
     );
-    assert(
-      wrapped.value.asBig.eq(info.value),
-      `Wrapped Big ${wrapped.value.asBig.toFixed()} did not match original value ${info.value.toFixed()}`
-    );
+    if (isNumber(info.input)) {
```

Comment:
so what are the type options for input here? numbers and what else?

Reply:
This is defined in `FixedInput`; currently we test MikeMcl's `Big`s and `BigNumber`s, ethers's `FixedNumber`s and `BigNumber`s, strings, `BN`s, `BigInt`s, and `number`s.

...that's odd, I don't see `Number`s.  Oops, guess I got mixed up there.  You think I should change it back to use `typeof`?

Reply:
which ones do we want this to apply to, just `number`s?

I'll leave it up to you, I'm not really sure

Reply:
Yeah, I think I just won't bother changing it...